src/app/admin/borrowers/borrowers.component.html

```diff
@@ -40,4 +37,66 @@ <h3>Borrowers</h3>
       </tbody>
     </table>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<!-- create borrower modal -->
+<ng-template #createBorrowerModal let-c="close" let-d="dismiss">
+  <form class="form-horizontal" #createBorrowerForm="ngForm"
```

Comment:
While the way you have it is the way it was in the example in the official documentation, it seems to me that it'd be better to have the `<form>` nested *inside* the modal's body, instead of surrounding the entire modal dialog.

Actually, it looks like you have a `<form>` there *inside* this one (line 57), so this should simply be removed (probably moving its `class` to the inner `<form>`).

Reply:
Yes, you are correct.